src/facilities/models.py

```diff
@@ -0,0 +1,171 @@
+import logging
+
+from django.core.exceptions import ValidationError
+from django.db import models
+from django.utils.text import slugify
+from utils.models import CampRelatedModel, UUIDModel
+
+logger = logging.getLogger("bornhack.%s" % __name__)
+
+
+class FacilityQuickFeedback(models.Model):
+    """
+    This model contains the various options for giving quick feedback which we present to the user
+    when giving feedback on facilities. Think "Needs cleaning" or "Doesn't work" and such.
+    This model is not Camp specific.
+    """
+
+    feedback = models.CharField(max_length=100)
+
+    icon = models.CharField(
+        max_length=100,
+        default="fas fa-exclamation",
+        blank=True,
+        help_text="Name of the fontawesome icon to use, including the 'fab fa-' or 'fas fa-' part. Defaults to an exclamation mark icon.",
+    )
+
+    def __str__(self):
+        return self.feedback
+
+
+class FacilityType(CampRelatedModel):
+    """
+    Facility types are used to group similar facilities, like Toilets, Showers, Thrashcans...
+    facilities.Type has a m2m relationship with FeedbackChoice which determines which choices
+    are presented for giving feedback for this Facility
+    """
+
+    class Meta:
+        unique_together = [("slug", "responsible_team")]
+
+    name = models.CharField(max_length=100, help_text="The name of this facility type")
+
+    slug = models.SlugField(
+        blank=True,
+        help_text="The url slug for this facility type. Leave blank to autogenerate one.",
+    )
+
+    description = models.TextField(help_text="Description of this facility type")
+
+    icon = models.CharField(
+        max_length=100,
+        default="fas fa-list",
+        blank=True,
+        help_text="Name of the fontawesome icon to use, including the 'fab fa-' or 'fas fa-' part.",
+    )
+
+    responsible_team = models.ForeignKey(
+        "teams.Team",
+        on_delete=models.PROTECT,
+        help_text="The Team responsible for this type of facility. This team will get the notification when we get a new FacilityFeedback for a Facility of this type.",
+    )
+
+    quickfeedback_options = models.ManyToManyField(
+        to="facilities.FacilityQuickFeedback",
+        help_text="Pick the quick feedback options the user should be presented with when submitting Feedback for a Facility of this type. Pick at least the 'N/A' option if none of the other applies.",
+    )
```

Comment:
Different facilities will share Feedback options. This can be a problem if the options are changed with the wrong assumption. Let's say there is the feedback "broken", this feedback can now changed to "clogged" and now a facility (like a lamp) can have this status.

On the other hand, I like that you would be able to filter everything with the status "broken", no matter if it's lamps or toilets.

I don't think this should change, just wanted to mention it :)

Reply:
Good point. I went back and forth a bit on the QuickFeedback, originally it was tied to the FacilityType and could not be reused, but it seemed like we would be repeating the same options like "Needs attention" and so on a lot. Maybe we could add a note to help_text for QuickFeedback or even a clean() method warning loudly when someone changes a QuickFeedback option which has already been used.

Reply:
Yes, go full in on this approach, that can make so much sense: More reuse and possibility to do overviews and statistics across different facility types. That approach will surely win over other approaches, like enum (non-db) values or purely unique options per facility.

Great idea with the `clean()` implementation and `help_text` for already used types!!